static/js/topic_page/map_tile.tsx

```diff
@@ -283,6 +283,7 @@ function draw(
     }
     return place.name + ": " + value;
   };
+  document.getElementById(props.id).innerHTML = "";
```

Comment:
nit: this shouldn't be required anymore -- please remove.

Reply:
@beets , the reason I added this was because before it, the map was getting duplicated multiple times (every time the state of the data got updated) but with the addition of this line, it's only being plotted once:)

Reply:
I see - let's do this within the disease page then (not in map_tile). It looks like we'll have to look at how the page is being redrawn with data refreshes.

Reply:
Done!